locale/en/knowledge/intermediate/how-to-log.md

```diff
@@ -57,17 +57,19 @@ Here is an example of setting up a `winston` logger.  This example includes most
 
       require('winston-riak').Riak;
       require('winston-mongo').Mongo;
+      require('winston-couchdb').Couchdb;
```

Comment:
Not sure we need the CouchDB adapter in this example, I'd rather mention it in the paragraph at the end (line 90).